src/CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ FILE(GLOB SOURCE_FILES
   "libs/lib.c"
   "views/view.c"
   )
-FILE(GLOB HEADER_FILES "*.h" "common/*.h" "external/CL/*.h" "control/*.h" "iop/*.h" "libs/*.h" "views/*.h")
+FILE(GLOB HEADER_FILES "*.h" "common/*.h" "external/OpenCL/CL/*.h" "control/*.h" "iop/*.h" "libs/*.h" "views/*.h")
```

Comment:
shouldnt this only be done if the headers are actually found? maybe in the If body in src/external/CMakeLists.txt.

Reply:
This is harmless if the submodule is not there as the globing won't do anything, no?